02-peer-protocol.md

```diff
@@ -926,6 +926,25 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
    * [`sha256`:`payment_hash`]
    * [`u32`:`cltv_expiry`]
    * [`1366*byte`:`onion_routing_packet`]
+   * [`update_add_htlc_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `update_add_htlc_tlvs`
+2. types:
+    1. type: 0 (`hold_htlc`)
+    2. data:
+        * [`32*bytes`:`payment_release_secret`]
+
+#### TLV fields for `held_htlc_available`
```

Comment:
Given `release_held_htlc` may be sent hours later, it seems weird that the sender has no way of knowing that `held_htlc_available` arrived successfully (I get that it's supposed to be sent over two different paths, but still). Should there be a `held_htlc_available_msg_received` reply?